channels/us.m3u

```diff
@@ -5490,17 +5490,17 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 https://a.jsrdn.com/broadcast/447912f76b/+0000/low/c.m3u8
 #EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ULF3iOE.png" group-title="Family",Positiv TV (1080p)
 https://api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ULF3iOE.png" group-title="Family",Positiv TV
 https://player-api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="Family",Positiv TV
 https://player-api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8?Faith
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="Family",Positiv TV
```

Comment:
Remove

Reply:
I can remove it. channel wasnt new. all are in the list now. So from now on if going through and testing you want me to remove working dup links that have the same url?